lib/Doctrine/ORM/Id/AssignedGenerator.php

```diff
@@ -59,12 +59,8 @@ public function generate(EntityManager $em, $entity)
             }
 
             if (isset($class->associationMappings[$idField])) {
-                if ( ! $em->getUnitOfWork()->isInIdentityMap($value)) {
-                    throw ORMException::entityMissingForeignAssignedId($entity, $value);
-                }
```

Comment:
hmm, indeed. The relation itslef will still be required to use a managed entity in other places